pkg/query-service/model/v3/v3.go

```diff
@@ -238,3 +238,17 @@ type Point struct {
 	Timestamp int64   `json:"timestamp"`
 	Value     float64 `json:"value"`
 }
+
+// ExploreQuery is a query for the explore page
+// It is a composite query with a source page name
+// The source page name is used to identify the page that initiated the query
+// The source page could be "traces", "logs", "metrics" or "dashboards", "alerts" etc.
+type ExplorerQuery struct {
+	UUID           string          `json:"uuid,omitempty"`
+	SourcePage     string          `json:"sourcePage"`
+	CompositeQuery *CompositeQuery `json:"compositeQuery"`
+	// ExtraData is JSON encoded data used by frontend to store additional data
+	ExtraData string `json:"extraData"`
+	// 0 - false, 1 - true
+	IsView int8 `json:"isView"`
```

Comment:
What is the use-case of `IsView`?

Reply:
>Possibility to save/load filters in Traces and Logs . This view has a lot of features. Just i would like to have possibility to save it . And reuse again later with actual timeframe

https://github.com/SigNoz/signoz/issues/1958
https://github.com/SigNoz/signoz/issues/2172